src/Classes/ServiceAPI/MyRadio_Season.php

```diff
@@ -403,6 +405,23 @@ public static function getForm()
                     'required' => false,
                 ]
             )
+        )->addField(
+            new MyRadioFormField(
+                'subtype',
+                MyRadioFormField::TYPE_SELECT,
+                [
+                        'options' => array_merge(
+                            [
+                            ['value' => '', 'text' => 'Leave unchanged']
+                            ],
+                            MyRadio_ShowSubtype::getOptions()
+                        ),
+                        'label' => 'Subtype',
+                        'explanation' => 'If necessary, override the subtype for this season. '
+                            .'If you\'re not sure what you\'re doing, leave it what it is.',
```

Comment:
Just a grammatical thingie, should probs be "leave it as it is"?

Reply:
Good point, updating